nntrainer/models/neuralnet.h

```diff
@@ -63,10 +63,12 @@ using DatasetModeType = ml::train::DatasetModeType;
  * @brief     Statistics from running or training a model
  */
 typedef struct RunStats_ {
-  float accuracy; /** accuracy of the model */
-  float loss;     /** loss of the model */
+  float accuracy;     /** accuracy of the model */
+  float loss;         /** loss of the model */
+  int count;          /** number of iterations done on this stat */
+  unsigned int right; /** number of right sample on this run */
```

Comment:
rename to `num_correct_predictions`? `right` is a bit vague.

Reply:
Done